### PR TITLE
Register VegaLite

### PR DESCRIPTION
**Pull request by fredo-dedup**
Provides bindings to the recently published Vega-Lite viz grammar

### CODE REVIEW

**Comment by tkelman** [comment]
Is any of the code derived from the existing Vega.jl package? If so it should contain a license attribution.

Also good idea to push the git tag to https://github.com/fredo-dedup/VegaLite.jl/releases so it can be easily cross-referenced while browsing the code.

**Comment by fredo-dedup** [comment]
Part of the code is indeed from the Vega.jl package (only the render.jl file). By license attribution, you mean adding `Copyright (c) 2016 John Myles White and Randy Zwitch.`  to the license file ?  Or is it more complicated ?

Should I also mention the 'University of Washington Interactive Data Lab.' (the copyright owners of Vega-Lite) ? They already have their own license file in the `asset/bower_components` directory, but I used their properties description in the package docs.

**Comment by fredo-dedup** [comment]
Latest version has the license attribution to the Vega.jl authors + this version has ben tagged.

File: VegaLite/versions/0.0.1/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.4
+JSON
```
Comment (tkelman) [inline VegaLite/versions/0.0.1/requires]: you added Compat here in the more recent tag

**Pull request merged**